README.md

```diff
@@ -78,6 +79,14 @@ Some quick links to useful parts of the documentation follow below for your conv
 
 To get started using rotki, check out the [detailed usage guide](https://rotki.readthedocs.io/en/latest/usage_guide.html).
 
+## Application Preview
+
+<div style="display: flex; justify-content: space-between">
```

Comment:
This looks really small currently @Narcos101. You can see it in your own fork. Can you make both pics take the max size and be the same size? https://github.com/Narcos101/rotki/tree/develop#application-preview